### PR TITLE
Remove 5.1 Bio-Formats jobs from the Contributing page

### PR DESCRIPTION
**Pull request by sbesson**
This development line is now unmaintained and the jobs are due for removal in
the upcoming months

### CODE REVIEW

**Comment by jburel** [comment]
Keep the B-F 5.1. docs jobs?

**Comment by sbesson** [comment]
Our policy with docs jobs is usually slightly different from the code jobs. Happy to clean the `ci-docs` page as well if we decide we are not maintaining these /cc @hflynn

**Comment by hflynn** [comment]
I can't see us doing a 5.1 docs release now, may as well clean those up too.

**Comment by mtbc** [comment]
Staged at http://www.openmicroscopy.org/site/support/contributing-staging/ci-bio-formats.html.

File: contributing/ci-bio-formats.txt [hunk 1]
```diff
@@ -7,216 +7,52 @@ All jobs are listed under the :jenkinsview:`Bio-Formats` view tab of Jenkins.
...
-                * :term:`BIOFORMATS-5.1-merge-test_images_good`
                 * :term:`BIOFORMATS-DEV-merge-test_images_good`
 
         -       * Runs automated tests against a subset of the data repository on squig
-                * :term:`BIOFORMATS-5.1-merge-repository-subset`
                 * :term:`BIOFORMATS-DEV-merge-repository-subset`
 
         -       * Runs openBytes performance test
-                * :term:`BIOFORMATS-5.1-merge-openbytes-performance`
                 * :term:`BIOFORMATS-DEV-merge-openbytes-performance`
```
Comment (mtbc) [inline contributing/ci-bio-formats.txt]: could get rid of extra line here but doesn't affect rendering

File: contributing/ci-bio-formats.txt [hunk 2]
```diff
@@ -444,3 +280,7 @@ Jenkins.
                    test by setting a space-separated list of formats for the
                    ``DEFAULT_FORMAT_LIST`` variable.
 
+        :jenkinsjob:`BIOFORMATS-DEV-merge-homebrew`
```
Comment (mtbc) [inline contributing/ci-bio-formats.txt]: this is a breaking job?

**Comment by mtbc** [comment]
Also staged at http://www.openmicroscopy.org/site/support/contributing-staging/ci-docs.html.

**Comment by mtbc** [comment]
Good to merge.

**Comment by hflynn** [comment]
Live contributing docs updated and I've also added the BF 5.1.x docs link to https://www.openmicroscopy.org/site/support/previous/ with a note that they will not be updated.

**Pull request merged**